src/lib/config.js

```diff
@@ -13,7 +13,11 @@ module.exports = {
   PORT: process.env.PORT || 5010,
   SESSION_SECRET: process.env.SESSION_SECRET,
   REDIS: {
-    SESSION_URL: process.env.REDIS_SESSION_URL,
+    SESSION_URL: process.env.REDIS_SESSION_URL || "localhost",
```

Comment:
This will disable the in-memory Redis, which is useful for standalone tests.

If you want to override this locally, you can put a REDIS_SESSION_URL=localhost in your `.env` file

Reply:
Nice pickup, I forgot to revert that 🤦🏻